orca-core/src/main/java/com/netflix/spinnaker/orca/pipeline/expressions/PipelineExpressionEvaluator.java

```diff
@@ -63,8 +64,11 @@
   private final ParserContext parserContext = new TemplateParserContext("${", "}");
   private final ExpressionsSupport support;
 
+  @Getter private final Set<String> executionAwareFunctions = new HashSet<String>();
```

Comment:
why do you need a `@Getter`?

Reply:
Just to access that in the test and verify it holds the correct functions.

Reply:
ah, right, thanks!